.gitlab-ci.yml

```diff
@@ -0,0 +1,94 @@
+deploy frankfurt dev:
+  variables:
+    KUBECONFIG: /home/gitlab-runner/.kube/frankfurt.kubeconfig
+  tags:
+    - aws
+    - frankfurt
+    - meao
+  only:
+    - dev
+    - frankfurt
+    - frankfurt-dev
+    - master
+  script:
+    - kubectl_1.11 apply -f frankfurt/snippets-dev
+    # --timeout requires kubectl 1.12+
+    - kubectl_1.11 rollout status -f frankfurt/snippets-dev/deploy.yaml -w
```

Comment:
After the push to dev triggered https://gitlab.com/mozmeao/snippets-config/-/jobs/225175689 I see that we need to modify this to call out specific deployments instead of just passing in the list. This approach worked with nucleus because there was only a single deployment in the file.